recipe-client-addon/bin/tc/fetch.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -eu
+
+# mach wants this
+export SHELL=$(which bash)
```

Comment:
I don't think we need this anymore. I added it because running mach in m-c fails if `$SHELL` is not set. I don't think we're using mach here, so we can remove this.